src/otx/recipes/stages/classification/incremental.yaml

```diff
@@ -29,7 +29,7 @@ lr_config:
   warmup_ratio: 0.333
 
 task_adapt:
-  type: "mpa"
+  type: "default_task_adapt"
```

Comment:
Wouldn't something like `enable: True` be better in this case, rather than using type separately?
What do you think?
It doesn't seem to fit with type.

Reply:
I'm not sure, since I don't have any background that why previous task adapt type is mpa. I just guess there could be another option for task_adapt. If there's not, maybe we can change type to enable. But it requires lots of changes, I want to make another PR for this and other things we discuss

Reply:
yeah, no problem